### PR TITLE
Update cartridge and metrics versions

### PR DESCRIPTION
**Pull request by yngvar-antonsson**
- [x] Tests
- [x] Changelog
- [x] Documentation

### CODE REVIEW

**Review by DifferentialOrange — APPROVED** [review]
It would be nice to put documentation and dependencies updates to separate commits, but this is fine too.

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## Unreleased
```
Comment (LeonidVas) [inline CHANGELOG.md:8]: [Unreleased]

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,11 @@ All notable changes to this project will be documented in this file.
...
+### Changed
```
Comment (LeonidVas) [inline CHANGELOG.md:9]: Add an empty line after ([Unreleased]).

**Review by LeonidVas — CHANGES_REQUESTED** [review]
Hi! Thank you for the patchset.
We are decorate the commit messages according to https://github.com/tarantool/tarantool/wiki/Code-review-procedure#commit-message
"Update cartridge and metrics versions" -> "app-dependency: update cartridge and metrics"
"Update getting-started-app docs" -> "documentation: update getting-started-app"

**Review by LeonidVas — APPROVED** [review]

**Pull request merged**